hbase-server/src/test/java/org/apache/hadoop/hbase/util/MultiThreadedAction.java

```diff
@@ -320,7 +320,7 @@ public boolean verifyResultAgainstDataGenerator(Result result, boolean verifyVal
    * @param verifyCfAndColumnIntegrity verify that cf/column set in the result is complete. Note
    *                                   that to use this multiPut should be used, or verification
    *                                   has to happen after writes, otherwise there can be races.
-   * @return
+   * @return the verified result from get or scan
```

Comment:
NIT, because it is a boolean it would be better to have it in the format "true if ...., false otherwise".

Reply:
@HorizonNet Thanks for your comment. Adressed it. Kindly have a look.